### ISSUE TITLE
Raspberry Pi (ARM64) support

### ISSUE THREAD

**Issue by TemporalAgent7** [billykwooten/ecobee-exporter#6]
This is looking great. It would be awesome if you could add support for arm64 images (for use on Raspberry Pi).

Thanks!

**Comment by mattmendick** [comment]
I agree, would be nice to have official support. But you can build your own, it just takes a bit.

First clone the project. Then you have to comment out this line in `Dockerfile` before it will work as it seems that the golang race detector doesn't work on the raspberry pi. Not totally clear on the issues here, but put a # before this line in `Dockerfile`:
```
RUN GO111MODULE=on CGO_ENABLED=1 go test -race -mod vendor ./...
```

Then you can run `./script/cibuild` from the root of the project, and after a bit you'll have a docker image. Mine was called `ecobee-exporter:60cfaa73e944`, yours will have a different hash at the end. Then I ran the docker command in the docs as normal, but replaced `billykwooten/ecobee-exporter` with `ecobee-exporter:60cfaa73e944`. You'll get this name output from the end of the build command.
